packages/fedi_app/lib/generated/intl/messages_ru.dart

```diff
@@ -212,33 +212,33 @@ class MessageLookup extends MessageLookupByLibrary {
             MessageLookupByLibrary.simpleMessage("Веб-сайт"),
         "app_about_learnMore": MessageLookupByLibrary.simpleMessage("Детали"),
         "app_about_title": MessageLookupByLibrary.simpleMessage("О Fedi"),
-        "app_acccount_my_customList_edit_account_action_add":
+        "app_account_my_customList_edit_account_action_add":
```

Comment:
Some projects don't like changing localizations. (Others request it, it's about 50/50, and I generally guess wrong.)